tests/Models/AddressStatus.php

```diff
@@ -16,7 +16,8 @@
 
 class AddressStatus extends Enum
 {
-    const __default = self::UNKNOWN;
+    const __DEFAULT = self::UNKNOWN;
+    const __default = self::UNKNOWN; // v2 default for backwards compatibility
```

Comment:
This approach (adding both constants) will increase ambiguity and can also give false positives.
In real-life circumstances, this scenario never happens, therefore I would not test against it.
I mean if we just fix the test for the sake of fixing then we're not going to have a reliable test suite.

Reply:
I get that, maybe we could remove the defaults in all the Enums except OrderStatus?

As far as I can see they aren't used in any tests, but I left them there since I couldn't tell if it was used on a deeper level.